Package.swift

```diff
@@ -0,0 +1,16 @@
+import PackageDescription
+
+let package = Package(
+    name: "SQLite",
+    targets: [
+        Target(
+            name: "SQLite",
+            dependencies: [
+                .Target(name: "SQLiteObjc")
+            ]),
+        Target(name: "SQLiteObjc")
+    ],
+    dependencies: [
+        .Package(url: "https://github.com/jberkel/CSQLite.git", majorVersion: 0)
```

Comment:
@stephencelis this should probably be moved to your namespace – I couldn't find a way to avoid a separate repository. there's something like dependency pinning now but it seems to be a newer feature, it's not on my system.